code/switch.js

```diff
@@ -0,0 +1,215 @@
+  1 - Delicious Pizza
+  2 - Pasta
+  3 - Salad`
+)
+)
+let selectedFood
+let subType
+let selectedSubType
+
+switch (foodChoice) {
+  case 1:
+  selectedFood = "delicious pizza"
+  alert(`I see! One ${selectedFood} coming up.`)
+  subType = parseInt( prompt(
+    `What kind of pizza would you prefer?
+    1 - Calzone
+    2 - Funghi
+    3 - Quattro Stagioni`
+  )
+  )
+    switch (subType) {
+      case 1:
+        selectedSubType = "Calzone"
+        alert(`I see you want to order a ${selectedFood} ${selectedSubType}, great choice`)
+        break
+        case 2:
+        selectedSubType = "Funghi"
+        alert(`I see you want to order a ${selectedFood} ${selectedSubType}, great choice`)
+        break
+        case 3:
+        selectedSubType = "Quattro stagioni"
+        alert(`I see you want to order a ${selectedFood} ${selectedSubType}, great choice`)
+        break
+        default:
+        alert("Choose a number to make an order")
+        exit(1)
+    }
+  break
+  case 2:
+  selectedFood = "pasta"
+  alert(`I see! One ${selectedFood} coming up.`)
+  subType = parseInt( prompt(
+    `What kind of pasta would you prefer?
+    1 - Carbonara
+    2 - Bolognese
+    3 - Tomato basilica`
+  )
+  )
+
+  switch (subType) {
+    case 1:
+      selectedSubType = "Carbonara"
+      alert(`I see you want to order a ${selectedFood} ${selectedSubType}, great choice`)
+      break
+      case 2:
+      selectedSubType = "Bolognese"
+      alert(`I see you want to order a ${selectedFood} ${selectedSubType}, great choice`)
+      break
+      case 3:
+      selectedSubType = "Tomato Basilica"
+      alert(`I see you want to order a ${selectedFood} ${selectedSubType}, great choice`)
+      break
+      default:
+      alert("Choose a number to make an order")
+      exit(1)
+  }
+  break
+  case 3:
+  selectedFood = "salad"
+  alert(`I see! One ${selectedFood} coming up.`)
+  subType = parseInt( prompt(
+    `What kind of salad would you prefer?
+    1 - Chicken
+    2 - Avocado
+    3 - Shrimp`
+  )
+  )
+
+  switch (subType) {
+    case 1:
+      selectedSubType = "with chicken"
+      alert(`I see you want to order a ${selectedFood} ${selectedSubType}, great choice`)
+      break
+      case 2:
+      selectedSubType = "with avocado"
+      alert(`I see you want to order a ${selectedFood} ${selectedSubType}, great choice`)
+      break
+      case 3:
+      selectedSubType = "with shrimp"
+      alert(`I see you want to order a ${selectedFood} ${selectedSubType}, great choice`)
+      break
+      default:
+      alert("Choose a number to make an order")
+      exit(1)
+  }
+  break
+default:
+  alert("Choose a number to make an order")
+  exit(1)
+}
+
+
+// Step 4 - Age
+// Your code goes here
+
+
+
+let portionSize
+let cost
+let confirmation 
+
+let age = parseInt( prompt(
+  `We serve both adult portion and child sized portion.
+
+Choose your preffered portion size
+by entering your age:`
+)
+)
+
+switch (true) {
```

Comment:
here as well in this statement, please use a formatter, it adds appropriate indentation and line breaks to improve readability ✨